src/main/java/com/minecolonies/coremod/inventory/api/CombinedItemHandler.java

```diff
@@ -38,27 +38,27 @@
     private       String                   customName;
 
     /**
-     * Method to create a new {@link AbstractCombinedItemHandler}.
+     * Method to create a new {@link CombinedItemHandler}.
      *
-     * @param defaultName The default name of this {@link AbstractCombinedItemHandler}.
+     * @param defaultName The default name of this {@link CombinedItemHandler}.
      * @param handlers    The combining {@link IItemHandlerModifiable}.
      */
-    protected AbstractCombinedItemHandler(@NotNull final String defaultName, @NotNull final IItemHandlerModifiable... handlers)
+    public CombinedItemHandler(@NotNull final String defaultName, @NotNull final IItemHandlerModifiable... handlers)
     {
         this.handlers = handlers;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Store a copy of "handlers". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2384)